scout/server/blueprints/cases/views.py

```diff
@@ -439,7 +387,14 @@ def phenotypes_actions(institute_id, case_name):
             store.remove_phenotype(institute_obj, case_obj, user_obj, case_url, hpo_id)
 
     if action == "ADDGENE":
-        hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        try:
+            hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        except ValueError:
+            flash(
+                "Provided gene info could not be parsed! " "Please allow autocompletion to finish.",
+                "warning",
+            )
+        # TODO: dont crash
```

Comment:
```suggestion
```

Reply:
That's always a good reminder, but I'll remove it.